packages/docusaurus-plugin-content-docs/src/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -791,8 +789,8 @@ Object {
                 \\"docId\\": \\"foo/baz\\"
               }
             ],
-            \\"collapsible\\": true,
-            \\"collapsed\\": true
+            \\"collapsed\\": true,
+            \\"collapsible\\": true
```

Comment:
This is very annoying, but Jest wants to reorder object keys in a snapshot even when the value doesn't change...